api/v1beta1/policyautomation_types.go

```diff
@@ -68,6 +81,14 @@ type PolicyAutomationList struct {
 	Items           []PolicyAutomation `json:"items"`
 }
 
+// PolicyAutomation events on each target cluster
+type ClusterEvent struct {
+	// Policy automation starts time as everyEvent mode
+	AutomationStartTime string `json:"automationStartTime"`
+	// The last time policy status from non-compliance to compliance
```

Comment:
Wouldn't the time here be the change from compliant to noncompliant?

Reply:
@gparvin  I agree here is a little weird and I will try to explain it:

In the first commit, I just update `eventTime` for the case that in the delay period, policy status change from non-compliant to compliant:
https://github.com/open-cluster-management-io/governance-policy-propagator/pull/38/commits/ce8fec06d0b5914d6670d9452fdd28e1d86ab6d2#diff-746b54415213bba767535a3153aff159b013aea7e7d74f6bcfeb0f135bd7c79cR270

Then next time when policy status changes from compliant to non-compliant again,  it will go to here:
https://github.com/open-cluster-management-io/governance-policy-propagator/pull/38/commits/ce8fec06d0b5914d6670d9452fdd28e1d86ab6d2#diff-746b54415213bba767535a3153aff159b013aea7e7d74f6bcfeb0f135bd7c79cR251

There are two ways to do this: 
A) just update `eventTime` for the event from non-compliant to compliant 
vs 
B)  updates`eventTime` for both events - non-compliant to compliant and compliant to non-compliant

If you run some test cases in mind, either A) or B) will work for the final result but B) may be easier to understand 
So I made another commit,  @gparvin @mprahl  you can comment on the second commit: https://github.com/open-cluster-management-io/governance-policy-propagator/pull/38/commits/1b2c18f6b93ec99ba17512fd60a0dd99103dd24c

@gparvin I'm not sure if your thought is the way C) just update `eventTime` for the event from compliant to non-compliant, but that way doesn't work for the multiple event case in the delay period. We can't tell the difference between the first time `compliant to non-compliant` and the second time `compliant to non-compliant` because we don't record all events details.

Reply:
@gparvin @mprahl Keeping two commits for easier reviewing, I will consolidate them later.

Reply:
Perfect, thanks Alex.  That was the additional clarity I wanted to confirm so this look great.

Reply:
@gparvin Thanks for reviewing, I just consolidated two commits.